apps/openmw/mwbase/soundmanager.hpp

```diff
@@ -14,6 +14,14 @@ namespace MWWorld
 
 namespace MWSound
 {
+    // Each entry excepts of MaxCount should be used only in one place
+    enum BlockerType
+    {
+        VideoPlayback = 0,
+
+        MaxCount = 1
```

Comment:
You don't need to explicitly declare values for enums, and in this case it's not recommended to. They always start at 0 and each entry is one more than the previous, so `MaxCount` is automatically correct.

Reply:
Can you guarantee that enums in C++ use indices on all platforms and old compilers?
It will be not a good thing to break, for example, Clang or Android builds because they use powers of two instead?

Reply:
[Positive](https://en.cppreference.com/w/cpp/language/enum):
"If the first enumerator does not have an initializer, the associated value is zero. For any other enumerator whose definition does not have an initializer, the associated value is the value of the previous enumerator plus one."